src/MoBi.Core/Serialization/ORM/Mappers/ProjectMetaDataToProjectMapper.cs

```diff
@@ -91,9 +93,18 @@ private void addNonProjectBuildingBlocks(List<IBuildingBlock> moduleBuildingBloc
          var simulationSettingsBlocks = moduleBuildingBlocks.OfType<SimulationSettings>().ToList();
          simulationSettingsBlocks.Each(x => moduleBuildingBlocks.Remove(x));
 
-         addModuleBuildingBlocks(moduleBuildingBlocks);
+
+         
+         if(hasUniqueBuildingBlocks(moduleBuildingBlocks))
+            addModuleBuildingBlocks(moduleBuildingBlocks);
+         else
+            addBuildingBlocksToModule(moduleBuildingBlocks);
```

Comment:
can you try to rename these two methods `addModuleBuildingBlocks` and `addBuildingBlocksToModule`. When I am reading along with the code, I can't determine what will be different.

maybe you can name one of them `addAllBuildingBlocksToModule` and one could be `addAllBuildingBlocksToSeparateModules`